app/design/frontend/base/default/template/securesubmit/form.phtml

```diff
@@ -138,8 +138,11 @@ SecureSubmitMagento.init({
 <?php if ($_loggedIn && $allow_card_saving && count($customerStoredCards)): ?>
 <script type="text/javascript">
 var secureSubmitsavedcardselector = document.querySelector('.saved-card-selector input');
-secureSubmitsavedcardselector.addEventListener('click', function(){
-    secureSubmitsavedcardselector.parentNode.classList.toggle('selected')
-}, false);
+    if(secureSubmitsavedcardselector){
+        secureSubmitsavedcardselector.addEventListener('click', function(){
+            secureSubmitsavedcardselector.parentNode.toggleClassName('selected');
```

Comment:
@ttnnkkrr Do Prototype helper methods get added when not using `$(...)` and `$$(...)` functions? Just want to ensure this doesn't error out